lib/src/builder.dart

```diff
@@ -93,7 +93,9 @@ class MockBuilder implements Builder {
           '// ignore_for_file: invalid_use_of_visible_for_testing_member\n'));
       b.body.add(Code('// ignore_for_file: prefer_const_constructors\n'));
       // The code_builder `asA` API unconditionally adds defensive parentheses.
-      b.body.add(Code('// ignore_for_file: unnecessary_parenthesis\n\n'));
+      b.body.add(Code('// ignore_for_file: unnecessary_parenthesis\n'));
```

Comment:
An aside for @srawlins – wonder if we should pull these ignores into a constant and DRY up this logic a bit...

Reply:
I do like having a comment for why each one might be needed.

Reply:
That could go in the constant definition, right?

Reply:
Yeah, it could.

 I do hope to elimitate all of these and replace with https://github.com/dart-lang/sdk/issues/46957.

Reply:
In the meantime, could we perhaps get this one merged? :)

Reply:
Yeah, the support just landed for "all lints", but it will be introduced in Dart 2.15, which is many moons from now.